### PR TITLE
INPC method rename

### PR DESCRIPTION
**Pull request by brianlagunas**
making API sonsistant with .NET coding standards and other Prism event
methods

### CODE REVIEW

File: Source/Prism.Tests/Commands/DelegateCommandFixture.cs [hunk 1]
```diff
@@ -374,7 +374,7 @@ public void NonGenericDelegateCommandObservingPropertyShouldRaiseOnEmptyProperty
 
             command.CanExecuteChanged += delegate { canExecuteChangedRaised = true; };
 
-            OnPropertyChanged(null);
+			RaisePropertyChanged();
```
Comment (bartlannoeye) [inline Source/Prism.Tests/Commands/DelegateCommandFixture.cs]: Using xamarin studio by any chance? Tabs vs spaces ;) (codebase is using spaces)

Comment (brianlagunas) [inline Source/Prism.Tests/Commands/DelegateCommandFixture.cs]: This is old code. I should have done a layout update :)

**Comment by dvorn** [comment]
@brianlagunas In 6.3 Release Notes these changes are listed under Prism for Xamarin.Forms while they belong to Prism.

**Comment by brianlagunas** [comment]
@dvorn thanks for the note. It's been fixed

**Pull request merged**